apps/examples/testcase/ta_tc/audio/itc/itc_audio_main.c

```diff
@@ -389,10 +950,10 @@ static void itc_audio_pcm_readi_tc_p(void)
 	int remain = 0;
 
 	g_pcm = pcm_open(0, 0, PCM_IN, NULL);
-	TC_ASSERT_GT("pcm_open", pcm_get_file_descriptor(g_pcm), 0);
+	TC_ASSERT_GT_CLEANUP("pcm_open", pcm_get_file_descriptor(g_pcm), 0, pcm_close(g_pcm));
 
 	buffer = malloc(pcm_get_buffer_size(g_pcm));
```

Comment:
pcm_get_buffer_size returns the no. frames in one buffer and not the size of buffer in bytes. please use pcm_frames_to_bytes function here to convert to bytes.